### DISCUSSION TITLE
a way to process inline markdown

### DISCUSSION THREAD

**Discussion by husayt** [antfu/vite-plugin-md#36] [Ideas]
Would be great to be able to convert markdown content received from api to html using this plugin, i,e, something like this:
```vue
hello this text {{  md("is **bold** ") }}  
```
Otherwise I have to seaprately import to `markdown-it` just for that purpose.

Thanks

**Comment by yankeeinlondon** [comment]
would something like this suffice in a markdown file:

```md
# My Document

## For Local Files
[load::inline](./README.md)

## For API Calls
[load::inline](https:/docs.somewhere.com, offset: 'data')

> where with the API call you can optionally take a property off the returned value
```

**Reply by yankeeinlondon** [reply]
The benefit of this syntax is that many editors will provide autocomplete for local files. No autocomplete for URLs but it's intuitive and consistent. This feature would have to be "opt-in" though.
